tests/e2e/online_tests.py

```diff
@@ -131,19 +151,34 @@ def _get_datasource(self, server_file):
 
     def _create_extract(self, type, wb_name):
         command = "createextracts"
-        arguments = [command, type, wb_name, "--encrypt"]
+        arguments = [command, type, wb_name]
+        if extracts_enabled:
+            arguments.append("--encrypt")
         _test_command(arguments)
 
     # variation: url
     def _refresh_extract(self, type, wb_name):
         command = "refreshextracts"
-        arguments = [command, type, wb_name]
-        _test_command(arguments)
+        arguments = [command, wb_name]  # should not need -w
+        try:
+            _test_command(arguments)
+        except Exception as e:
+            print(e)
+            print("expected (tabcmd classic)")
+            print("  *** Unexpected response from the server: Bad request")
+            print("This refresh extracts operation is not allowed for workbook World Indicators (errorCode=80030)")
 
     def _delete_extract(self, type, item_name):
         command = "deleteextracts"
         arguments = [command, type, item_name, "--include-all"]
-        _test_command(arguments)
+        try:
+            _test_command(arguments)
+        except Exception as e:
+            print(e)
+            print("Expected (tabcmd classic:")
+            print("*** Unexpected response from the server: Unable to load Data Source")
```

Comment:
I think both these commands should work in classic, if we have a valid workbook.

Reply:
Do you want me to remove the try catch blocks then? How certain are you that these will pass with tabcmd classic. I can run a test pass using tabcmd classic if you want me to double check.

Reply:
Hm, not that certain. But if it doesn't pass, I'd like to look into that.

Reply:
For now I guess just leave the try-catch blocks